README.md

```diff
@@ -87,8 +87,8 @@ The following 44 devices are supported.
 | [PCD8544 display](http://eia.udg.edu/~forest/PCD8544_1.pdf) | SPI |
 | [Resistive Touchscreen (4-wire)](http://ww1.microchip.com/downloads/en/Appnotes/doc8091.pdf) | GPIO |
 | [Semihosting](https://wiki.segger.com/Semihosting) | Debug |
-| [Shift register](https://en.wikipedia.org/wiki/Shift_register#Parallel-in_serial-out_\(PISO\)) | GPIO |
-| [Shift Registers Serial In Parallel Out (SIPO)](https://en.wikipedia.org/wiki/Shift_register#Serial-in_parallel-out_(SIPO)) | GPIO |
+| [Shift register (PISO)](https://en.wikipedia.org/wiki/Shift_register#Parallel-in_serial-out_\(PISO\)) | GPIO |
+| [Shift Registers (SIPO)](https://en.wikipedia.org/wiki/Shift_register#Serial-in_parallel-out_(SIPO)) | GPIO |
```

Comment:
Really tiny detail and I think we are good to merge this, but the lines are different _register_ != _Registers_, please modify so they look alike:

[Shift register (PISO)]
[Shift register (SIPO)]

Reply:
Also, remove these lines https://github.com/simulot/drivers/blob/shiftregister/shiftregister/shiftregister.go#L3-L8